flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/parser/TransformParser.java

```diff
@@ -95,6 +95,8 @@ public class TransformParser {
     private static final Logger LOG = LoggerFactory.getLogger(TransformParser.class);
     private static final String DEFAULT_SCHEMA = "default_schema";
     private static final String DEFAULT_TABLE = "TB";
+    private static final String MAPPED_COLUMN_NAME_PREFIX = "field";
```

Comment:
Just thinking if we can use RegExp-like names (`$1`, `$2`, ...)? Ignore this comment if it doesn't look right.